frontend/src/model/Component.ts

```diff
@@ -1,5 +1,5 @@
 export type Component = {
     category: string
     type: string
-    ageKm: number
-}
\ No newline at end of file
+    ageKm: number | undefined
```

Comment:
In my opinion "ageKm" is a little bit confusing. But maybe it is just for me, because I'm not that much into your project and into bikes in general.